docs/riskAuthentication.md

```diff
@@ -58,25 +58,27 @@ If the `Device Profiling Method` setting in your adapter configuration is set to
 
 ### Setup
 
-If the `Device Profiling Method` setting in your adapter configuration is set to `Captured by a previous adapter` and `Device Profiling Script Source` is set to `ThreatMetrix SDK`:
+If the `Device Profiling Method` setting in your adapter configuration is set to `Use existing ThreatMetrix session ID` and `Device Profiling Script Source` is set to `ThreatMetrix SDK`:
 1. Copy the `tmx_sdk_profiling.js` file from the integration `.zip` file to a location that your application can access.
 2. Add the following to the sign-on page. Adjust the path and file name of the script file.
 ```html
 <script type="text/javascript" src="tmx_sdk_profiling.js"></script>
 <script type="text/javascript">pinghelper.run("<deviceProfilingDomain>", "<orgId>");</script>
 ```
-If the `Device Profiling Method` setting in your adapter configuration is set to `Captured by a previous adapter` and `Device Profiling Script Source` is set to `ThreatMetrix Web`:
+If the `Device Profiling Method` setting in your adapter configuration is set to `Use existing ThreatMetrix session ID` and `Device Profiling Script Source` is set to `ThreatMetrix Web`:
 1. Copy the `tmx_web_profiling.js` file from the integration `.zip` file to a location that your application can access.
 2. Add the following to the sign-on page. Adjust the path and file name of the script file.
 ```html
 <script type="text/javascript" src="tmx_web_profiling.js"></script>
 ```
-If the `Device Profiling Method` setting in your adapter configuration is set to `Captured by this adapter` and `Device Profiling Script Source` is set to `ThreatMetrix SDK`:
+If the `Device Profiling Method` setting in your adapter configuration is set to `Create new device profile` and `Device Profiling Script Source` is set to `ThreatMetrix SDK`:
 1. Copy the `tmx_sdk_profiling.js` file from the integration `.zip` file to a location that your application can access.
 2. Where your web application initializes `PfAuthnWidget`, adjust the path (`deviceProfileScript`) to the script file.
 ```javascript
 var authnWidget = new PfAuthnWidget('https://localhost:9031', { divId: 'authnwidget', deviceProfileScript: './tmx_sdk_profiling.js' });
 authnWidget.init();
 ```
-If the `Device Profiling Method` setting in your adapter configuration is set to `Captured by this adapter` and `Device Profiling Script Source` is set to `ThreatMetrix Web`, no extra steps are needed.
+If the `Device Profiling Method` setting in your adapter configuration is set to `Create new device profile` and `Device Profiling Script Source` is set to `ThreatMetrix Web`, no extra steps are needed.
+
+> **_Note:_** the labels for the choices for `Device Profiling Method` were updated as of ThreatMetrix Kit 1.2.
```

Comment:
Should I leave the old label values here as well?

Reply:
Might be best if to check with Mike/Tom, but I think it might be better if we have two sets of instructions, one for 1.1 and another for 1.2 and later (like how some PF instruction differs between diff PF versions).

Reply:
Will ask them.

Reply:
I asked Mike and he said

> we're all good with just the 1.2 labels. The change in terms is covered in the changelog, and that keeps out GitHub instructions much cleaner.